vignettes/plotting_vignette.Rmd

```diff
@@ -0,0 +1,160 @@
+---
+title: "Plotting Functionality in tcplfit2"
+author: "Center for Computational Toxicology and Exposure"
+output:
+   prettydoc::html_pretty:
+    theme: architect
+    toc: yes
+    toc_depth: 4
+vignette: >
+  %\VignetteIndexEntry{Plot_with_tcplfit2}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, packages, warning=FALSE, message=FALSE}
+library(tcplfit2)
+library(dplyr)
+library(stringr)
+library(ggplot2)
+library(gridExtra)
+```
+
+```{r, include=FALSE}
+source("~/CompTox-ToxCast-tcplFit2/R/concRespPlot2.R")
+source("~/CompTox-ToxCast-tcplFit2/R/plot_allcurves.R")
+```
+
+
+# Introduction
+This vignette demonstrates how to plot concentration-response modeling curves with two `ggplot2 based` plotting functions in `tcplfit2`: `plot_allcurves` and `concRespPlot2`. 
```

Comment:
* remove either 'modeling' in this sentence - or you can replace 'curves' with 'results', either fix will work
* the quotes around "ggplot2 based", suggest only doing them around "ggplot2" 
* instead of ":" suggest using ", namely"